src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/BomEditMessageBuilder.java

```diff
@@ -90,6 +90,9 @@ public String getNotificationType() {
         BlackDuckResponseCache responseCache = new BlackDuckResponseCache(bucketService, blackDuckBucket, timeout);
         BomEditNotificationContent bomEditContent = notificationView.getContent();
         Optional<ProjectVersionComponentView> bomComponent = responseCache.getBomComponentView(bomEditContent.getBomComponent());
+
+        // TODO Stop using this when Black Duck supports going back to the project-version
+        // blackduck-common-api 2019.12.0.4+ should have the project version link in the BomEditNotificationContent
```

Comment:
Moving the comment here from `BlackDuckResponseCache ` because the comment only applies to the Bom Edit notifications